modules/perception/map/hdmap/hdmap_input.cc

```diff
@@ -163,9 +163,8 @@ void HDMapInput::MergeBoundaryJunction(
   road_boundaries_ptr->resize(polygon_size);
   int polygons_index = 0;
   // merege boundary
-  int step = hdmap_sample_step_;
   PointDCloudPtr temp_cloud = base::PointDCloudPool::Instance().Get();
-  for (int i = 0; i < polygon_size; i++) {
+  for (int step, i = 0; i < polygon_size; i++) {
```

Comment:
It would be nicer if you define step at line 174(or 175) when you define step variable

Reply:
Thanks for you comments and yep, that's would be more readable.